dbms/tests/queries/0_stateless/00910_client_window_size_detection.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+function cleanup() { rm -f -- -; }
```

Comment:
Why the file is named `-`? Is it necessarily?

Reply:
It is created by the `script`, hm, but I guess I can write something like to avoid this `-` -- `script ... /dev/null`

Reply:
Fixed this by using `/dev/null`